packages/design-system/src/components/MonthPicker/MonthPicker.stories.tsx

```diff
@@ -9,27 +13,79 @@ const meta: Meta<typeof MonthPicker> = {
     inversed: false,
   },
   argTypes: {
+    selectedMonths: { control: 'multi-select', options: monthNumbers },
+    defaultSelectedMonths: { control: 'multi-select', options: monthNumbers },
+    disabledMonths: { control: 'multi-select', options: monthNumbers },
     errorMessage: { control: 'text' },
     hint: { control: 'text' },
     requirementLabel: { control: 'text' },
   },
+  render: function Component(args) {
+    if (args.selectedMonths?.length === 0) {
+      args.selectedMonths = undefined;
+    }
+    if (args.defaultSelectedMonths?.length === 0) {
+      args.defaultSelectedMonths = undefined;
+    }
+    if (args.disabledMonths?.length === 0) {
+      args.disabledMonths = undefined;
+    }
+    return <MonthPicker {...args} key={JSON.stringify(args.defaultSelectedMonths)} />;
+  },
 };
 export default meta;
 
 type Story = StoryObj<typeof MonthPicker>;
 
+const ControlledTemplate: Story = {
+  render: function Component(args) {
+    const [{ selectedMonths }, updateArgs] = useArgs();
+    const onChange = (event) => {
+      action('onChange')(event);
+      const month = parseInt(event.currentTarget.value, 10);
+      const newSelectedMonths = selectedMonths.slice();
+      if (newSelectedMonths.includes(month)) {
+        newSelectedMonths.splice(newSelectedMonths.indexOf(month), 1);
+      } else {
+        newSelectedMonths.push(month);
+      }
+      updateArgs({ selectedMonths: newSelectedMonths });
+    };
+
+    const onSelectAll = () => {
+      const newSelectedMonths = monthNumbers.filter((m) => !args.disabledMonths?.includes(m));
+      updateArgs({ selectedMonths: newSelectedMonths });
+    };
+
+    const onClearAll = () => {
+      updateArgs({ selectedMonths: [] });
+    };
+
+    return (
+      <MonthPicker
+        {...args}
+        selectedMonths={selectedMonths}
+        onChange={onChange}
+        onSelectAll={onSelectAll}
+        onClearAll={onClearAll}
+      />
+    );
+  },
+};
+
 export const Default: Story = {
+  // ...UncontrolledTemplate,
```

Comment:
What happened to the uncontrolled template?

Reply:
oh... moved it to the render in the meta data like we did for Pagination stories. Forgot to remove this comment.

Reply:
oh okay, I think the commit diff I was looking at was missing the fact that it moved to the meta section. I thought it was lost.